tests/dhcp_relay/test_dhcp_relay_stress.py

```diff
@@ -214,6 +216,111 @@ def _verify_client_packets(pkts):
             ptfhost.shell('rm -f {}'.format(count_file))
 
 
+@pytest.mark.parametrize('dhcp_type', ['discover', 'offer', 'request', 'ack'])
+def test_dhcpcom_relay_counters_stress(ptfhost, ptfadapter, dut_dhcp_relay_data, validate_dut_routes_exist,
+                                       testing_config, setup_standby_ports_on_rand_unselected_tor,
+                                       toggle_all_simulator_ports_to_rand_selected_tor_m,     # noqa F811
+                                       dhcp_type, clean_processes_after_stress_test,
+                                       rand_unselected_dut):
+    '''
+    Test DHCP relay counters functionality can handle the maximum load within 5% miss.
+    '''
+    testing_mode, duthost = testing_config
+    packets_send_duration = 120
+    client_packets_per_sec = 50
+    if duthost.hostname.__contains__('720dt'):
```

Comment:
Better to use hwsku dict like below
https://github.com/sonic-net/sonic-mgmt/blob/master/tests/pc/test_lag_member.py#L45